internal/pkg/node/constructors.go

```diff
@@ -31,6 +31,8 @@ defaultnode:
   init: /sbin/init
   root: initramfs
   ipxe template: default
+  grub template: grub.cfg.ww
+  boot method: ipxe
```

Comment:
This is great (I was requesting a generic "boot method" or "bootloader" option, and a per-node setting, earlier; but is this actually used anywhere? I don't see `boot method` anywhere else in the code.